lieferemma_api.proto

```diff
@@ -74,6 +77,18 @@ enum Currency {
     EUR = 978;
 }
 
+// Supported Measurement Packaging units for your product 
+enum Unit {
+    KG = 0;
```

Comment:
I'm wondeing if we could e.g. SI units, but not sure if there is some kind of list that does not totally blow up and only contains relevant units.

Reply:
Hmm, do not think that that is possible. I think our current list is enough. As packaging can be described in so many different ways. I changed some away from SI Unit so we can use uint32 instead of double.